dagql/testdata/introspection.json

```diff
@@ -636,17 +636,7 @@
             "args": [],
             "isDeprecated": false,
             "deprecationReason": "",
-            "directives": [
-              {
-                "name": "impure",
-                "args": [
-                  {
-                    "name": "reason",
-                    "value": "Because I said so."
-                  }
-                ]
-              }
-            ]
```

Comment:
I never saw these used in the first place (so seems good to remove them), but also, mildly curious as to why we had an `impure` directive at all :thinking:

Reply:
No concrete reason, it was just interesting to surface at the schema layer because it could affect how things use those fields. For example, the CLI used to fetch object fields and print them; it could avoid fetching `@impure` fields as a safety precaution. But now that feature's gone, and it never used it to begin with.